.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
+
+on:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - id: release-notes-1
+        name: "Release notes for latest version"
+        uses: ./
+        with:
+          file: ./test/CHANGELOG.md
+
+      - id: release-notes-2
+        name: "Release notes for an explicit version"
+        uses: ./
+        with:
+          file: ./test/CHANGELOG.md
+          version: 2.2.0.0
+
+      - id: release-notes-3
+        name: "Release notes for an explicit version (with v-prefix)"
+        uses: ./
+        with:
+          file: ./test/CHANGELOG.md
+          version: v2.2.0.0
+
+      - id: release-notes-4
+        name: "Release notes for non-existent version"
+        uses: ./
+        with:
+          file: ./test/CHANGELOG.md
+          version: v42.42.42
+
+      - name: Assertions
```

Comment:
This is really cool. So we run the action with different arguments, it saves the results, and then we can diff to see if the results match our fixtures? Sweet!!